installation/cmd/run.ps1

```diff
@@ -1,25 +1,17 @@
 param (
     [string]$CR_PATH = "",
     [switch]$SKIP_MINIKUBE_START = $false,
-    [switch]$KNATIVE = $false,
     [string]$VM_DRIVER = "hyperv",
     [string]$PASSWORD = ""
 )
 
 $CURRENT_DIR = Split-Path $MyInvocation.MyCommand.Path
 $SCRIPTS_DIR = "${CURRENT_DIR}\..\scripts"
 $DOMAIN = "kyma.local"
-$INSTALLER_EXTRA_ARGS = ""
-$MINIKUBE_EXTRA_ARGS = ""
 $CREATE_CR_EXTRA_ARGS = ""
 
-if ($KNATIVE -eq $true) {
-    $MINIKUBE_EXTRA_ARGS = "${MINIKUBE_EXTRA_ARGS} -memory 10240 -disk_size 30g"
-    $INSTALLER_EXTRA_ARGS = "${INSTALLER_EXTRA_ARGS} -knative"
-}
-
 if ($SKIP_MINIKUBE_START -eq $false) {
-    Invoke-Expression -Command "${SCRIPTS_DIR}\minikube.ps1 -vm_driver ${VM_DRIVER} -domain ${DOMAIN} ${MINIKUBE_EXTRA_ARGS}"
+    Invoke-Expression -Command "${SCRIPTS_DIR}\minikube.ps1 -vm_driver ${VM_DRIVER} -domain ${DOMAIN}

```

Comment:
Missing " at the end

Reply:
I will create a second PR for this. To unblock other tasks, I will merge it once the tests succeed.